src/openmethane_prior/layers/omAgLulucfWasteEmis.py

```diff
@@ -27,10 +27,10 @@
 import pyproj
 import rioxarray as rxr
 import xarray as xr
-from omInputs import domainProj, livestockDataPath, sectoralEmissionsPath, sectoralMappingsPath
-from omInputs import domainXr as ds
-from omOutputs import convertToTimescale, landuseReprojectionPath, sumLayers, writeLayer
-from omUtils import area_of_rectangle_m2, secsPerYear
+from ..omInputs import domainProj, livestockDataPath, sectoralEmissionsPath, sectoralMappingsPath
+from ..omInputs import domainXr as ds
+from ..omOutputs import convertToTimescale, landuseReprojectionPath, sumLayers, writeLayer
+from ..omUtils import area_of_rectangle_m2, secsPerYear
```

Comment:
nit: This is a personal preference, but I prefer absolute imports. If we refactor our tools are smart enough to also handle tweaking updating things everywhere

```suggestion
from openmethane_prior.omInputs import domainProj, livestockDataPath, sectoralEmissionsPath, sectoralMappingsPath
from openmethane_prior.omInputs import domainXr as ds
from openmethane_prior.omOutputs import convertToTimescale, landuseReprojectionPath, sumLayers, writeLayer
from openmethane_prior.omUtils import area_of_rectangle_m2, secsPerYear
```

Reply:
You could probably do a global find and replace of `from ..` to `from openmethane_prior.`